### PR TITLE
App 97 grouped response

### PR DESCRIPTION
**Pull request by tarikstafford**
## [APP-97] && [APP-329]

- Number action set and text action set - created an internal model to reduce copy pasta
- Added grouped response
- Made text bubbles a bit fatter - maybe I fed them too much?

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

[APP-97]: https://hedvig.atlassian.net/browse/APP-97
[APP-329]: https://hedvig.atlassian.net/browse/APP-329

### CODE REVIEW

File: Projects/Embark/Sources/MessageBubble.swift [hunk 1]
```diff
@@ -15,23 +15,38 @@ struct MessageBubble {
     let animationDelay: TimeInterval
     let animated: Bool
     let messageType: MessageType
+    let pills: [String]
 
-    init(text: String, delay: TimeInterval, animated: Bool = false, animationDelay: TimeInterval = 0, messageType: MessageType = .received) {
+    init(text: String, delay: TimeInterval, animated: Bool = false, animationDelay: TimeInterval = 0, messageType: MessageType = .received, pills: [String] = []) {
```
Comment (sampettersson) [inline Projects/Embark/Sources/MessageBubble.swift:20]: NIT: Split up row

File: Projects/Embark/Sources/MessageBubble.swift [hunk 2]
```diff
@@ -48,9 +63,10 @@ extension MessageBubble: Viewable {
         stylingView.alpha = 0
 
         let containerView = UIStackView()
+        containerView.axis = .vertical
         containerView.isLayoutMarginsRelativeArrangement = true
         containerView.insetsLayoutMarginsFromSafeArea = false
-        containerView.layoutMargins = UIEdgeInsets(top: 5, left: 15, bottom: 3, right: 15)
+        containerView.layoutMargins = UIEdgeInsets(top: 12, left: 15, bottom: 12, right: 15)
```
Comment (sampettersson) [inline Projects/Embark/Sources/MessageBubble.swift:69]: This spacing looks insane 😅 I think we need to make it smaller

**Review by sampettersson — CHANGES_REQUESTED** [review]
Codewise this looks good! I noticed that the NumberActionSet component disappears off-screen before the animation is complete also.

File: Projects/Embark/Sources/MessageBubble.swift (hunk 2 above)
Comment (tarikstafford) [reply Projects/Embark/Sources/MessageBubble.swift:69]: > Codewise this looks good! I noticed that the NumberActionSet component disappears off-screen before the animation is complete also.

I couldn't replicate this... hmm

**Comment by tarikstafford** [comment]
@sampettersson I ended up resigning first responder before running the animation loop for translating the action set component down and this looks okay, what are your thoughts? --- I couldn't figure out why it was accelerating down but anyways.

**Review by sampettersson — APPROVED** [review]
Any reason why upstream is `moving-flow` here btw? Couldn't this go straight to main? :)

**Pull request closed without merging**